src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -694,6 +694,10 @@ fun getPartition(entityKeyId: UUID, partitions: List<Int>): Int {
     return partitions[entityKeyId.leastSignificantBits.toInt() % partitions.size]
 }
 
+fun getPartition(entityKeyId: UUID, partitions: Array<Int>): Int {
```

Comment:
Doesn't something like this already exist in PostgresDataQueries?

Reply:
Yes, but it takes a list instead of a typed array. I guess could have routed it to list version by doing .asList

Reply:
oh oops I didn't see this, but why is it being converted to a typed array at all?

Reply:
I had to do a conversion from set and I could gone to list or array.  I did array for reason mentioned above.

Reply:
I feel like this and the above `getPartition` should both be in `PartitionManager` instead of here in `PostgresDataQueries` but I don't think there being two methods is an issue

Reply:
Done